src/index.html

```diff
@@ -66,15 +51,16 @@ <h1 class="title head-title">Leader Board</h1>
                 <div class="field">
                   <div class="control">
                     <input
+                      id="score"
                       class="input-width input"
-                      type="text"
+                      type="number"
                       placeholder="Your Score"
                     />
                   </div>
                 </div>
                 <div class="field is-grouped is-align-content-center">
                   <div class="control">
-                    <button class="button is-link" type="button">Submit</button>
+                    <button class="button is-link" type="button" id="submit">Submit</button>
```

Comment:
- [ ] The submit button should hit the `POST API` endpoint and post user entries, your submit button does not post the user entry, please check the `postData` function and make sure it works.